docs/tutorials/tutorials_intro.md

```diff
@@ -1,3 +1,19 @@
 ## Arm Mbed tutorials
 
-The Arm Mbed ecosystem is expansive and offers many opportunities. In contrast to other sections of the documentation, which provide background reference material, this section provides step-by-step instructions for specific tasks you may wish to perform. These include Blinky, our example application; applications that build on Blinky; and debugging tasks. There are also tutorials about using the Arm Mbed Online Compiler, optimizing memory and optimizing power.
+The Arm Mbed ecosystem is expansive and offers many opportunities. In contrast to other sections of the documentation, which provide background reference material, this section provides step-by-step instructions for specific tasks you may wish to perform. These include our quick start guide, example applications that build on it and debugging tasks. There are also tutorials about using the Arm Mbed Online Compiler, optimizing memory and optimizing power.
+
+The [Mbed OS quick start](mbed-os-quick-start.html) is our quick start guide, a tutorial that shows you how to set up and run an example application.
+
+[Major releases](major-releases.html) discusses the difference between Mbed OS 2 and Mbed OS 5 and answers frequently asked questions about using the two major versions of Mbed OS. This section also teaches you how to migrate projects to Mbed OS 5.
```

Comment:
Makes me think this doesn't belong in the tutorials. Maybe move it to the new porting area?

Reply:
We had it in porting once upon a time, but there was a comment about how we were mixing porting concepts and migrating Mbed 2 to Mbed OS 5 is not the same as porting targets, so they don't belong in the same section.

Reply:
It probably doesn't belong in the intro, either.